/**:
  ros__parameters:
    ## Location calculation method
    # Options: "SinglePointDepth", "HostSpatials"
    coordinate_calc_method: "SinglePointDepth"

    ## HostSpatials choices
    # Options: "bbox_center", "bbox_roi"
    host_spatials_method: "bbox_center"

    # Averaging neighboring pixels' depth for depth of a point
    neighbourhood_pixels: 4

    # In meters
    thresh_high_depth: 7.000
    thresh_low_depth: 0.500

    clip_z: True